test/Microsoft.AspNetCore.Razor.Evolution.Test/IntegrationTests/BasicIntegrationTest.cs

```diff
@@ -97,8 +97,8 @@ public void Process_CustomDirective()
             Assert.Equal("test_directive", directiveSpan.Content);
 
             var irDocument = document.GetIRDocument();
-            var irNamespace = irDocument.Children[0];
-            var irClass = irNamespace.Children[0];
```

Comment:
I think we could just remove this test, baselining is a better strategy for this anyway.